README.md

```diff
@@ -197,3 +207,16 @@ Object with the default options for the notifications
 
 ## defaultOptionsByType
 Object with the default options for the notifications for specific types
+
+## Building
+
+_OS X users, ensure you have `wget` installed (available in `brew`)_
+
+To run the tests:
+
+```bash
+$ ./workaround.sh
```

Comment:
@hiddentao Very weird. what is the console output you get when running the tests?
Just: ``` meteor test-packages ./``` from the package root should be enough